en/config_vtol/vtol_back_transition_tuning.md

```diff
@@ -1,57 +1,19 @@
 # VTOL Back-transition Tuning
 
 When a VTOL performs a back-transition (transition from fixed wing mode to multicopter) it needs to slow down before the multicopter can take proper control.
```

Comment:
> The big part of that was removing the docs about reversing thrust during backtransition, which is a feature that we no longer support. The new way of having the vehicle pitch up to break proved to be sufficient for all the vehicles I know. And it comes with less hardware complexity and safety risk.

@sfuhrer Thanks for the detailed explanation - makes sense.

Just a though, but it seems to me that some people might wonder **how** you back transition, and possibly why you don't use reverse thrust.

Do you think it is worth adding a note above this line "like":

> VTOL supports back transition to multicopter using vehicle pitch-up to brake.
> This has proved sufficient for all vehicle types tested.

Reply:
Currently I kind of covered that with

> To help with breaking, the controller will pitch up the vehicle if the current deceleration is below what is set in expected deceleration ([VT_B_DEC_MSS]

But please feel free to add your comment, it's correct.

Reply:
It is sufficient ... and yet I missed it. I've proposed an alternative in https://github.com/PX4/PX4-user_guide/pull/2422 that might be OK.